pom.xml

```diff
@@ -3,7 +3,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>com.createsend</groupId>
   <artifactId>createsend-java</artifactId>
-  <version>5.1.4-SNAPSHOT</version>
+  <version>6.0.0-SNAPSHOT</version>
```

Comment:
Will the release artifact contain `-SNAPSHOT` in its version?

Reply:
shouldn't be if following https://github.com/campaignmonitor/createsend-java/blob/master/RELEASE.md
the release prepare and perform will remove the SNAPSHOT version